### PR TITLE
use underscore view for file list upload

### PR DESCRIPTION
**Pull request by veolys-nanne**
[OO-MISC] Use view for file list in upload context

### CODE REVIEW

File: MediaAdminBundle/Resources/public/coffee/view/MediaUploadView.coffee [hunk 1]
```diff
@@ -54,64 +54,27 @@ MediaUploadView = OrchestraView.extend(
 
     @r.on 'fileAdded', (file) ->
       $('.flow-progress, .flow-list').show()
-
-      progressText = viewContext.renderTemplate('OpenOrchestraMediaAdminBundle:BackOffice:Underscore/Include/uploadProgress',
-        uniqueIdentifier: file.uniqueIdentifier
+      file.uploadProgressView = new OpenOrchestra.Upload.UploadProgressView(
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/coffee/view/MediaUploadView.coffee]: can you use app view configurator

File: MediaAdminBundle/Resources/public/coffee/view/UploadProgressView.coffee [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    'click .flow-file-pause': 'filePause'
+    'click .flow-file-resume': 'fileResume'
+    'click .flow-file-cancel': 'fileCancel'
+
+  ###*
+   * initialize
+   * @param {Object} options
+  ###
+  initialize: (options) ->
+    @options = options
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/coffee/view/UploadProgressView.coffee]: can you use reduce option or specify options in jsdoc

**Comment by veolys-nanne** [comment]
update done @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**